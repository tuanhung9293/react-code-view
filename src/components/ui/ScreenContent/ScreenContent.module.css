.base{
  box-sizing: border-box;
  padding: 0 20px;
  display: flex;
  flex-direction: column;
  flex: 1 0 auto;
  width: 100%;
  max-width: 1400px;
  margin: 0 auto;
}

@media (max-width: 670px) {
  .base{
    padding: 10px;
  }
}