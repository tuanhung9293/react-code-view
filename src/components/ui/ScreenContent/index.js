import React from 'react'
import styles from './ScreenContent.module.css'

function ScreenContent(props) {
  return <div {...props} className={styles.base} />
}

export default ScreenContent