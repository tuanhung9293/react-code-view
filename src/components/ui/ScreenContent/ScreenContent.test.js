import React from 'react'
import {shallow} from 'enzyme'
import ScreenContent from './'


describe('renders', () => {
  it('renders ScreenContent', () => {
    expect(shallow(
        <ScreenContent />
    )).toMatchSnapshot()
  })

  it('ScreenContent should pass props down', () => {
    expect(shallow(
        <ScreenContent data1="data 1" data2="data 2" />
    )).toMatchSnapshot()
  })
})