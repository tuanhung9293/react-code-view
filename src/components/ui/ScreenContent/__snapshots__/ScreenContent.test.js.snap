// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ScreenContent should pass props down 1`] = `
<div
  className="base"
  data1="data 1"
  data2="data 2"
/>
`;

exports[`renders renders ScreenContent 1`] = `
<div
  className="base"
/>
`;
